meetings/2024/2024-09-24.md

```diff
@@ -0,0 +1,47 @@
+## Agenda
+- Review Actions from last meeting see also https://github.com/orgs/datafoodconsortium/projects/5/views/
+- Semantizer / streams
+- Data capture
+- Connector codegen PR's
+- Hacktoberfest?
+- Define next meeting date: 08.10.2024 4pm CET
+
+## Reminders
+
+- Modularizing our ontology and version control: update from the ontology circle
+- Releases: Do we want a release schedule (e.g. 1 major per year, 1 minor per month) - we need to contribute to this discussion: https://github.com/orgs/datafoodconsortium/discussions/35
+
+## Participants
+
+- @RaggedStaff
+- @jgaehring
+- @lecoqlibre
+- @RachL
+
+## Connector codegen PR's and Data capture 
+
+Two PR need review and were added to the board by Garethe:
+
+https://github.com/datafoodconsortium/connector-codegen/pull/20 and https://github.com/datafoodconsortium/connector-codegen/pull/23
+
+These PR are based on the old and unstable version of the connector and semantizer. A new version of the semantizer will be release on September 29th.
```

Comment:
I think it was just the semantizer that was an issue, not the connector. @lecoqlibre can you confirm?

Reply:
Yes.